lib/driver.js

```diff
@@ -332,20 +335,18 @@ class XCUITestDriver extends BaseDriver {
       await this.stopRemote();
     }
 
-    await this.runReset();
+    if (this.opts.resetOnSessionStartOnly === false) {
```

Comment:
pretty sure we can go with the more succinct `!this.opts.resetOnSessionStartOnly`?